wit-0.3.0-draft/monotonic-clock.wit

```diff
@@ -0,0 +1,47 @@
+package wasi:clocks@0.3.0;
+/// WASI Monotonic Clock is a clock API intended to let users measure elapsed
+/// time.
+///
+/// It is intended to be portable at least between Unix-family platforms and
+/// Windows.
+///
+/// A monotonic clock is a clock which has an unspecified initial value, and
+/// successive reads of the clock will produce non-decreasing values.
+@since(version = 0.2.0)
```

Comment:
A little unsure if this version should remain 0.2.0 as that is when this interface was introduced or if the expectation is rev this to 0.3.0 now that functions on this interface return types only available in 0.3.0.

Reply:
The `since` version should only indicate what it is backwards-compatible with. Since this 0.3.0 will make breaking changes to the 0.2.x series, the version here should be updated to 0.3.0.

Reply:
Ok easy enough. Since the ABI is breaking, I will rev every since to 0.3.0.